### PR TITLE
Gateway Intents Guide

### PR DESCRIPTION
**Pull request by ShiroNotFriday**

### CODE REVIEW

**Comment by ShiroNotFriday** [comment]
Let me know if attachments don't work. Due to the current weak internet, I was not able to check their valid display on the page and so on.

**Comment by noituri** [comment]
The embedded videos do not work

**Review by MineBartekSA — CHANGES_REQUESTED** [review]
Aside from fixing the videos, please add information about the `Message Content Intent`

File: src/guides/gatewayIntents.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Gateway Intents
+This guide will tell you how to use Gateway Intents in BDFD, and what they are for.
```
Comment (MineBartekSA) [inline src/guides/gatewayIntents.md:2]: `This guide will tell you` isn't wrong, but `This guide will explain to you` or `In this guide, you will learn` would be better.

**Comment by ShiroNotFriday** [comment]
I will add attachments tomorrow. I'm out of city right now, so my poor internet connection doesn't allow me to add my attachments properly. I waited over an hour for them to upload to the resource, but... no need to explain.

**Review by MineBartekSA — COMMENTED** [review]
For now, looks good.
We'll wait for you to add the gifs

**Review by MineBartekSA — APPROVED** [review]
Thanks for your hard work

**Pull request merged**